test/parallel/test-events-once.js

```diff
@@ -24,18 +24,16 @@ async function onceAnEvent() {
   strictEqual(ee.listenerCount('myevent'), 0);
 }
 
-async function onceAnEventWithNullOptions() {
+async function onceAnEventWithInvalidOptions() {
   const ee = new EventEmitter();
 
-  process.nextTick(() => {
-    ee.emit('myevent', 42);
-  });
-
-  const [value] = await once(ee, 'myevent', null);
-  strictEqual(value, 42);
+  await Promise.all([1, 'hi', null, false].map((options) => {
```

Comment:
```suggestion
  await Promise.all([1, 'hi', null, false, () => {}, Symbol(), 1n].map((options) => {
```

Reply:
Updated test case. Thank you for enhancement.